aiida_common_workflows/workflows/relax/abinit/generator.py

```diff
@@ -420,3 +433,67 @@ def generate_inputs_calculation(
     }
 
     return dictionary
+
+
+def get_initial_magnetization(structure: StructureData) -> List[float]:
+    """Generate a guess for initial magnetization using a magnetic moment mapping.
+
+    See the function for elements with known magnetization guesses. If a gues is not known, 0.01 is used
+    for initialization.
+
+    :param structure: structure
+    :return: scalar initial magnetization for each Site in the structure
+    """
+
```

Comment:
Please add a reference where you got the initial magnetization values.

Reply:
Added a note that the values are from `abipy` and `aiida-quantumespresso`